generators/next-js/templates/circleci.yaml.ejs

```diff
@@ -49,11 +49,15 @@ jobs:
           path: ~/project
       - attach_workspace:
           at: ~/project
+      - run:
+          command: yarn install --prod --frozen-lockfile
       - run:
           name: Create archive
           command: |
             tar --create --gzip --file=archive.tar.gz --owner=0 --group=0 \
               build/ \
+              next.config.js \
```

Comment:
Since the config contains the path to the built directory, Next needs it to find the built files (that's why it can't be a typescript file and has to be pure javascript).